README.md

```diff
@@ -138,13 +166,39 @@ TBD
   - Updating the COWorkflowsBasePlugin.
   - Including gnomAD exomes and genomes for nocontrol workflow filtering
 
-* Version update to 1.2.177
+* Version update to 1.2.177-8
 
-  - Roddy 3.0 support.
-  - 1.2.177 is equivalent to 1.0.176-9.
-  - 1.2.177-1 is equivalent to 1.0.176-10.
+* Version update to 1.2.177-7
 
-* Version update to 1.0.176-9
+* Version update to 1.2.177-601 (ReleaseBranch_1.2.177-6)
+
+  - Bugfix: Fix of column swap bug introduced before 1.0.167. Output VCF with swapped control and tumor genotype columns if they are in the 11th and 10th column respectively.
+
+* Version update to 1.2.177-6
```

Comment:
These changes are in 1.2.177-5

Reply:
The bugfix is what you are just about to create, right? And it is what is described here. There is a ReleaseBranch_1.2.177-6, that actually also starts with 1.2.177-6, not with the previous tag.

So in the moment I don't understand your comment.

Reply:
Ah, right, I see what you mean.